app/src/main/java/com/hedvig/app/feature/webonboarding/WebOnboardingActivity.kt

```diff
@@ -95,10 +93,23 @@ class WebOnboardingActivity : BaseActivity(R.layout.activity_web_onboarding) {
                 else -> "/no"
             }
 
+            val encodedToken = URLEncoder.encode(getAuthenticationToken(), UTF_8)
+            val encodedQuoteID = URLEncoder.encode(intent.getStringExtra(QUOTE_ID), UTF_8)
+
             when (marketProvider.market) {
-                Market.NO -> webOnboarding.loadUrl(
-                    "${BuildConfig.WEB_BASE_URL}$webPath/start?variation=android#token=$encodedToken"
-                )
+                Market.NO -> {
+                    val isOffer = intent.getBooleanExtra(OFFER, false)
+                    if (isOffer) {
+                        webOnboarding.loadUrl(
+                            "${BuildConfig.WEB_BASE_URL}/no/new-member/offer?variation=android&quoteId=$encodedQuoteID#token=$encodedToken"
```

Comment:
We might want to place something like a `TODO` here or some other reminder that this might not be the final method for us accessing the Offer-page 🙂

Reply:
Sorry had the wrong base branch, this should not be included in this PR. Changed base!